ormconfig.json

```diff
@@ -0,0 +1,55 @@
+[
```

Comment:
Why did you delete the `ormconfig.js` file? The reason we were using it was to be able to pass credentials as environment variables in for production credentials, something we can't do with a `json` file. I suggest you declare your test object in the exported object of `ormconfig.js` file. Another important thing to note is credentials, even if they are test credentials are *never* committed. That's why we were using the `dotenv-safe` library to pass in the credentials as `env` variables. This way, it's not only secure, but offers the team flexibility in setting up their dev environments (e.g not everyone will have `admin` and `admin123` as username credentials).

Reply:
sounds good. WIP